docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: '3.6'
+
+services:
+    postgres:
+        image: postgres:9.6-alpine
+        ports:
+            - 5432:5432
+        environment:
+            - POSTGRES_USER=sqalx
+            - POSTGRES_PASSWORD=sqalx
+
+    mysql:
+        # image: mysql:8.0 # intel only
+        image: mysql/mysql-server:8.0 # mac M1 preview
```

Comment:
I'd do it the other way around unless this image works in all platforms.

Reply:
yes, sorry - artifact of me testing on my Mac M1 :-D